airflow/providers/alibaba/provider.yaml

```diff
@@ -21,6 +21,7 @@ name: Alibaba
 description: |
     Alibaba Cloud integration (including `Alibaba Cloud <https://www.alibabacloud.com//>`__).
 
+suspended: false
```

Comment:
I decided to add mandatory "suspended" in provider.yaml as it will be much more explicit and easier to discover that you actually "can" suspend a provider.